.github/workflows/Coverage.yml

```diff
@@ -0,0 +1,28 @@
+name: uvwasi coverage
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+    steps:
+      - uses: actions/checkout@v1
+      - name: Use Node.js ${{ matrix.node-version }}
```

Comment:
We shouldn't need Node.js at all for this job. Same for the environment info below, since that should be reported from the main workflow already.